AgentDiscoveries-Backend/src/test/java/org/softwire/training/utils/PagerTest.java

```diff
@@ -56,6 +56,15 @@ void calculatesEndPage() {
         assertEquals(10, endPage);
     }
 
+    @Test
+    void startPageIsFivePagesBeforeCurrentPageAfterPageFive() {
+        pager = new Pager(200, 6, 10);
```

Comment:
FWIW, I'd pick 7 instead of 6, so that the start page isn't 1 (in theory someone could slip up, make it start six pages previous, and then this test would still pass because of your other edge condition)

I'm sure you're sick of listening to me now 😉 so once that's sorted, you can merge it straight in